requirements.txt

```diff
@@ -10,3 +10,8 @@ requests >=2.23.0
 pytz >=2011k
 httplib2
 lxml
+pytest ==7.3.1
+allure-pytest ==2.9.45
+pytest-nose-attrib >=0.1.1
```

Comment:
I mean it's a good idea to lock all versions, not just pytest.

Reply:
fixed